rs/nns/governance/src/neuron_store.rs

```diff
@@ -702,16 +731,33 @@ impl NeuronStore {
     pub fn has_neuron_with_account_id(&self, account_id: &AccountIdentifier) -> bool {
         self.get_neuron_id_for_account_id(account_id).is_some()
     }
-
-    pub fn active_neurons_iter(&self) -> impl Iterator<Item = &Neuron> {
-        self.heap_neurons.values()
+    pub fn with_active_neurons_iter<R>(
+        &self,
+        callback: impl for<'b> FnOnce(Box<dyn Iterator<Item = Neuron> + 'b>) -> R,
+    ) -> R {
+        if self.use_stable_memory_for_all_neurons {
```

Comment:
How will this work during migration, when some active neurons are still in heap?